src/sentry/conf/server.py

```diff
@@ -1937,6 +1937,19 @@ def build_cdc_postgres_init_db_volume(settings):
     "sentry-swift": "https://docs.sentry.io/clients/cocoa/",
 }
 
+# Some of the links below are not ideal, but that is all migration documentation we currently have and can provide at this point
+SDK_MIGRATION_URLS = {
```

Comment:
what's the point of defining this as a separate dictionary? The way you wrote the code, doesn't this mean you will never link to any `/clients/foo/` link if there's a migration guide, right?

Reply:
I just wanted to keep it separated but you're right